src/splitInteger.test.js

```diff
@@ -4,18 +4,36 @@ const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
+    let result = splitInteger(6, 2); 
```

Comment:
You need to provide "expect" statements here.  for example: expect(splitInteger(8, 4)).toEqual([2, 2, 2, 2]); No new variable, not any other structures.

Reply:
Done